fetchAllEnv/fetchAllEnv.go

```diff
@@ -30,23 +30,25 @@ import (
 )
 
 type EnvField struct {
-	Env            string
-	EnvValue       string
-	EnvDescription string
+	Env               string
+	EnvValue          string
+	EnvDescription    string
+	EnvPossibleValues string
```

Comment:
new field->  `type`  (to be derived from dataType of struct
rename  `EnvPossibleValues` -> `example`
 envDescription -> description

generated filename should be user-friendly